docs/converter/supported_ops.md

```diff
@@ -0,0 +1,93 @@
+# Supported Ops
+## Converter Ops with Limitations
+### Base Limitations
+- **Output**
+    - Requires each output channel size <= `1024`.
+
+### Blueoil Customized Ops
+- **QTZ_binary_channel_wise_mean_scaling**
+- **QTZ_binary_mean_scaling**
+- **QTZ_linear_mid_tread_half**
+
+### Tensorflow Supported with Limitations
+- **[tf.layers.AveragePooling2D](https://www.tensorflow.org/versions/r1.15/api_docs/python/tf/layers/AveragePooling2D)**
+    - Currently, support only `2D`.
+    - **Do not** support `kernel depth = 1`.
+- **[tf.concat](https://www.tensorflow.org/versions/r1.15/api_docs/python/tf/concat)**
+    - **Do not** support concat of mixed data types (e.g., quantized values and float values).
+    - If inputs are quantized, requires `Each input channel size = multiple of 32`.
+- **[tf.layers.Conv2D](https://www.tensorflow.org/versions/r1.15/api_docs/python/tf/layers/Conv2D)**
+    - Support only convolution `2D`.
+    - **Do not** support transpose.
+    - Requires `kernel size = 1x1` or `3x3` or `5x5`.
+    - Requires `Input channel size = multiple of 32`, otherwise zero padding is used.
+    - If output is quantized by later operations, `Output channel size = multiple of 32`, otherwise output channel size is free from limitation (but performance will be worse).
+- **[tf.nn.depth_to_space](https://www.tensorflow.org/versions/r1.15/api_docs/python/tf/nn/depth_to_space)**
```

Comment:
Sorry, what is the kernel for s2d?

Reply:
My understanding is the kernel size in block_size for either s2d or d2s.

Reply:
Then how about to write `block_size ` instead of `kernel`.

Reply:
Fixed